src/haddock/modules/analysis/rmsdmatrix/rmsd.py

```diff
@@ -0,0 +1,265 @@
+"""RMSD calculations."""
+import os
+from pathlib import Path
+
+import numpy as np
+
+from haddock import log
+from haddock.libs.libontology import PDBFile
+from haddock.modules.analysis.caprieval.capri import get_atoms
+
+
+class RMSDJob:
+    """A Job dedicated to the fast rmsd calculation."""
+
+    def __init__(
+            self,
+            output,
+            params,
+            rmsd_obj):
+
+        log.info(f"core {rmsd_obj.core}, initialising RMSD...")
+        log.info(f"core {rmsd_obj.core}, # of pairs : {rmsd_obj.npairs}")
+        self.output = output
+        self.params = params
+        self.rmsd_obj = rmsd_obj
+        log.info(f"core {rmsd_obj.core}, RMSD initialised")
+
+    def run(self):
+        """Run this RMSDJob."""
+        log.info(f"core {self.rmsd_obj.core}, running RMSD...")
+        self.rmsd_obj.run()
+        self.rmsd_obj.output()
+        return
+
+
+class RMSD:
+    """RMSD class."""
+
+    def __init__(
+            self,
+            model_list,
+            core,
+            npairs,
+            start_ref,
+            start_mod,
+            output_name,
+            path,
+            **params,
+            ):
```

Comment:
It makes sense. I didn't do it previously because at some point the majority of these functions will be merged into a unique `geometry` library. But the class declaration deserves more, I agree :)

Reply:
Yes. When you merge the functions to a specific library, you can re-use the docstrings as well.